components/Cards/examples/ShapedCardViewController.swift

```diff
@@ -0,0 +1,225 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+
+@available(iOS 9.0, *)
+class CardView: MDCCard {
```

Comment:
Aside: do you expect it to be common to subclass MDCCard?

I'm assuming the answer is yes, but was also curious if you'd considered making a custom content view and adding that to a plain MDCCard's contentView as a subview.

Reply:
I think both options are viable options. I personally prefer subclassing in this case because masking the content of the MDCCard's subviews to fit the shape made the most sense for me to be done in `layoutSubviews`. If we were to create a custom subview then we would either have to reach its superview (the card) to get the shapeLayer in its `layoutSubviews` or set the mask in the view controller itself.

Therefore I do expect MDCCard to be subclassed, and I think it is based on the developer's preference and the specific use case.

Reply:
Solid answer. Thank you!